scripts/recovery_sync

```diff
@@ -40,4 +40,5 @@ done
 rsync -a /etc/machine-id etc/
 rsync -a --relative /export/home/delphix/.ssh .
 
-find . -print0 | cpio --null --create --format=newc | gzip -7 >"$target"
+find . -print0 | cpio --null --create --format=newc | gzip -7 >"$target.tmp"
```

Comment:
may we leave `$target.tmp` around in the event of a failure? if so, is that OK?

Reply:
Ideally we'd remove it; I'll add a cleanup function